More new lines for translation

Knofbath opened the pull request.

Commit:
More new lines for translation

Commit:
Update binary_translations.pm

Maybe F is too wide? Attempt to not truncate "Escort Fleet" to "Escort Flee"

"Transport ships" shrunk to "Transports"

Commit:
Update binary_translations.pm

More Compass Fairy strings
Quick Refuel/Reload
Battle Commands
Air Superiority strings (probably all images)

miyagami committed:
adding some more, not all, translations.

added more translations to list. some are listed with an [OC], which means they are Out of Context. I would like to see them in actual use before i commit to them, or would change them better. Welcome to the problems of localizing.
also corrected the testosterone mistake.

miyagami committed:
Update binary_translations.pm

Commit:
Update binary_translations.pm

Commit:
Moved Compass fairy down in code

miyagami committed:
Update csharp_translations.pm

miyagami committed:
Updating some translations better.

Updating some translations better.

Commit:
Update binary_translations.pm

miyagami committed:
More clarifications and substitutions

Keep in mind, I am intentionally waiting on : えいえいえーいっ and とまれ一つ from the Compass Fairy. I would like to see them in context as she is like a child, using Hiragana where Kanji would be most helpful.

miyagami committed:
Some more Compass Fairy thoughts

Commit:
Update binary_translations.pm

Commit:
Had them right way around the first time

Commit:
Compass Fairy identification returned

Commit:
Discord images added for each of the 4 fairies

miyagami committed:
Update binary_translations.pm

After the shown video, I have changed my stance on some of these. 
Important thing about translating, "Context Matters".

miyagami commented:
> > ※艦娘保有数が上限に近いため、新しい艦娘と邂逅できない可能性があります。 - "※If you have too many Ship Girls, there is a possibility of not getting a Ship Girl reward." - Edited for consistency.
> 
> Commie and the anime dub both call them Fleet Girls

Good for them.

They are called KanMusu(me)
Which, technically means "Ship Daughter"

艦 [Kan] is Ship
艦隊 [Kantai] is Fleet

Also, there are pre-established terms used in the game as is. Some I will be editing. Some I will not. At current, the translation "Ship Girl(s)" stands.

miyagami committed:
translation editing

Updated some translations to match the previous teams work. 
\resources\Textures\info6_set.tex.png ** these are final translations between 接近 and 統射

Commit:
More lines added

miyagami committed:
this is supposed to be a side project.

Just more translations. As usual, some are more localized than others.
Brevity.

Commit:
One more, and translation fixes

miyagami commented:
Good catch. I will wait to see how it works in game before i say anything about it. 
However, again, context matters.

miyagami committed:
This sounds better

Based on Mithaldu and my discussions.

wchristian commented:
I've integrated all the changes and comments from this.

You might however still see some translations missing if the patcher couldn't uniquely and confidently verify their location in the binary files. Please let me know if this is the case, probably best by using either of these:

https://github.com/wchristian/kc-vita-translation/issues
https://github.com/wchristian/kc-vita-translation/pulls
